student.rb

```diff
@@ -1,8 +1,10 @@
 require_relative 'person'
 
 class Student < Person
+  belongs_to :classroom
+
   include Person
-  attr_accessor :classroom
```

Comment:
- [ ] I think you deleted it but I highly recommend add getter here for the classroom since you already added a method the classroom, so you should add getter to it so you can add `attr_reader :classroom` 🚀🔥✅